imod_coupler/drivers/dfm_metamod/mf6_wrapper.py

```diff
@@ -66,16 +66,21 @@ def get_river_stages(
         stage = bound[:, 0]
         return stage
 
-    def get_river_flux(
+    def get_river_flux_estimate(
         self,
         mf6_flowmodel_key: str,
         mf6_river_pkg_key: str,
     ) -> NDArray[np.float_]:
         """
-        returns the river fluxes consistent with current river head, river stage and conductance.
+        returns the river1 fluxes consistent with current head, river stage and conductance.
         a simple linear model is used: flux = conductance * (stage - max(head, bot))
         Bot is the levelof the bottom of the river.
 
+        This function does not use the HCOF and RHS for calculating the flux, bacause it is used
+        at the beginning of the timestep, after updating the river stage by dflow. At that time
+        the package HCOF and RHS are not updated yet by MF6. Therefore we use the bottom level,
+        conducatnce and head of the previous timestep, and the stage of the new timestep.
+
```

Comment:
it's nice you renamed the function to "get_river_flux_estimate" but the description you provide here is totally specific to our use-case. How about "This function computes the flux based on the river stage, the river bottom elevation and the stored hydraulic head. This can be different from the flux computed by the river package itself if the river stage was set, but the timestep has not been solved by modflow"